crates/rpc-types-eth/src/transaction/receipt.rs

```diff
@@ -138,7 +138,7 @@ impl<T> TransactionReceipt<T> {
 #[doc(alias = "AnyTxReceipt")]
 #[cfg(feature = "serde")]
 pub type AnyTransactionReceipt =
-    alloy_serde::WithOtherFields<TransactionReceipt<alloy_consensus::AnyReceiptEnvelope<Log>>>;
+    alloy_serde::WithOtherFields<TransactionReceipt<alloy_consensus_any::AnyReceiptEnvelope<Log>>>;
```

Comment:
make sense to move this receipt too right?

Reply:
ah you're right. moved it to `alloy-rpc-types-any` 🙏 
https://github.com/alloy-rs/alloy/pull/1609/commits/5b5d2ac2fd4044b1e63ba5fa52a17d5b46ab54d5